src/sw/Core/Inc/Utils.h

```diff
@@ -39,6 +39,11 @@ enum Color
 	GRAY = 0X8430,
 };
 
-
+enum ChooseSensor
```

Comment:
We should have sth like a pin number, not really enum. Only if every next sensor has a different pin, like not the next one. Then we should discuss a better option.